.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.6.0",
-  "packages/api-explorer": "0.9.7",
-  "packages/code-editor": "0.1.1",
-  "packages/extension-api-explorer": "21.6.0",
+  ".": "1.7.0",
+  "packages/api-explorer": "0.9.8",
+  "packages/code-editor": "0.1.2",
+  "packages/extension-api-explorer": "21.7.0",
   "packages/extension-sdk": "21.6.0",
-  "packages/extension-sdk-react": "21.6.0",
-  "packages/hackathon": "21.6.0",
-  "packages/run-it": "0.9.7",
+  "packages/extension-sdk-react": "21.7.0",
+  "packages/hackathon": "21.7.0",
+  "packages/run-it": "0.9.8",
   "packages/sdk": "21.6.0",
-  "packages/sdk-codegen": "21.0.16",
-  "packages/sdk-codegen-scripts": "21.0.16",
+  "packages/sdk-codegen": "21.1.0",
+  "packages/sdk-codegen-scripts": "21.1.0",
```

Comment:
I'm working on a feature in [release-please](https://github.com/googleapis/release-please/pull/903) that will allow us to configure a "patch-only" auto-version-bump strategy for these looker-versioned packages. But, until that lands we still need to tweak `release-please-config.json` to manually set:

`packages/{extension-api-explorer,extension-sdk-react,hackathon}` need to have a `"release-as": "21.6.1"`
`packages/{sdk-codegen,sdk-codegen-scripts}` need to have a `"release-as": "21.0.17"`